index.js

```diff
@@ -7,12 +7,17 @@
 
 'use strict';
 
+const finite = Number.isFinite ? Number.isFinite : isFinite
+
 module.exports = function(num) {
   if (typeof num === 'number') {
     return num - num === 0;
   }
   if (typeof num === 'string' && num.trim() !== '') {
-    return Number.isFinite ? Number.isFinite(+num) : isFinite(+num);
+    return finite(+num);
+  }
+  if (typeof num === 'bigint') {
+    return finite(Number(num))
```

Comment:
Why convert? why not `true`?

Reply:
the "finite" function (Number.isFinite) will check whether the output is a value that can be represented by a Javascript number. The return value will be a Boolean.

Reply:
I'm sorry, I wasn't clear. I meant that `BigInt`s could always be considered as "valid numerical values" (as in math), not necessarily "all `BigInt`s are valid `Number`s, because they never coerce to `NaN`"

Reply:
Wait, now that I think about it, that should be a separate function named `isNumeric`

Reply:
Yeah, probably.
Anyway I don't think this changeset is getting merged (almost three years old 🍰)